### PR TITLE
Adds django 2.1 support (via install and test)

### PR DESCRIPTION
**Pull request by jheld**
Adds support for django 2.1, via installation and test suite.

#139

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -22,7 +22,7 @@
         'twilio>=6.3.0,<7',
         'Django>=1.8,<1.9;python_version=="3.3"',
         'Django>=1.8,<2;python_version<"3.4"',
-        'Django>=1.8,<2.1;python_version>="3.4"',
+        'Django>=1.8,<=2.1;python_version>="3.4"',
```
Comment (ryanhiebert) [inline setup.py]: I don't think this does what you're intending. This is the same as saying `<=2.1.0`, so it won't work with security releases. If you're wanting to relax it to allow the latest `2.1.X`, you probably want `<2.2`.

Comment (jheld) [reply setup.py]: Totally right! Thank you. I definitely meant `<2.2`

**Comment by jheld** [comment]
@ryanhiebert I agree, django doesn't explicitly expect perfect runs (if at all) for some of those releases. I've updated again. Unfortunately it looks like travis-ci is working feverishly to get py3.7 support, but it's still not prime time.

And yes, thank you for your willingnes to run tests against your project.

My company has recently upgraded to py36, so going forward py3 compatibility runs should be easier for me to test, aside from just the unit test suite.

**Comment by jheld** [comment]
Excellent. @ryanhiebert I will merge & release tonight.

**Comment by jheld** [comment]
Merged and deployed to PyPI.

**Pull request merged**